messages.py

```diff
@@ -0,0 +1,130 @@
+import random
+
+"""
+    Template credit: nomoreracistcops.github.io
+    Additional demands from:
+    https://secure.everyaction.com/eR7GA7oz70GL8doBq19LrA2
+    https://docs.google.com/document/d/1V2l7bdg6QeqtqVWDr5HFzVzieiT0vg6jZASN6syBNyY/edit
+"""
+
+# Randomly generates the subject header of the email
+def gen_subject():
+    s = ["Human Rights Inquiry", "Thoughts of a Concerned Citizen", "In Light of Recent Human Rights Abuses", "The Need for Police Oversight", "The Need for Police Accountability", "The Failures of Modern Law Enforcement", "Law Enforcement Must Change", "The Voice of a Troubled Citizen", "The Need for Law Enforcement Reform", "Reforms to Law Enforcement Needed", "Your Duty as a Public Servant"]
+    return random.choice(s)
+
+# Randomly generates the body of the email, follows structure of template and swaps out select words/phrases
+def gen_body(src_name, dst_name, location):
+    return "%s\t%s\t%s\t%s" % (gen_greeting(dst_name), gen_intro(location), gen_curiosity(), gen_conclusion(src_name))
+
+# Generates the greeting to the recipient of the email
+def gen_greeting(person):
+    s = ["Dear", "Hello", "Greetings", "Hi"]
+    return "%s %s,\n\n" % (random.choice(s), person)
+
+# Prepends greeting statement to a user-generated message
+def attach_greeting(person, body):
+    s = ["Dear", "Hello", "Greetings", "Hi"]
+    return "%s %s,\n\n%s" % (random.choice(s), person, body)
```

Comment:
What's the point of attach_greeting, if we can either call gen_greeting independently/are already call gen_greeting in the gen_body method?

Reply:
No real point was prob just for ease of use. This is just code I took from the other repo, I haven't made any modifications, was more putting this up as an example for what we could do.